lib/fx/statements/function.rb

```diff
@@ -29,7 +29,10 @@ module Function
       #     $$ LANGUAGE plpgsql;
       #   SQL
       #
-      def create_function(name, version: 1, sql_definition: nil)
```

Comment:
I wonder if it's enough to use https://github.com/ruby/ruby2_keywords instead?

Reply:
Seems that doesn't  not fixes the errors, also print warnings: 
```
/home/oscar/workspace/fx/lib/fx/statements/function.rb:32: warning: Skipping set of ruby2_keywords flag for create_function (method accepts keywords or method does not accept argument splat)
```
I have used it this way:
```ruby
ruby2_keywords def create_function(name, version: 1, sql_definition: nil)
# rest of code...
```